.github/workflows/build-coop.yaml

```
@@ -97,131 +73,31 @@ jobs:
-                tar -xzf SDL2.tar.gz
-                cd SDL2-2.30.9
-                ./configure --prefix=/usr/local CC="clang -arch x86_64 -mmacosx-version-min=10.15"
-                make -j$(sysctl -n hw.ncpu)
-                make install
+              if: ${{ matrix.plat.name != 'Windows' }}
+              run: ${{ matrix.plat.deps }}
 
             - name: Build the game
-              run: |
-                gmake OSX_BUILD=1 -j$(sysctl -n hw.ncpu)
+              run: ${{ matrix.plat.make }} ${{ case(matrix.plat.name == 'SteamOS', 'HANDHELD=1', '') }}
```

actionlint (1.7.12)

[error] property "plat" is not defined in object type {platform: any}

(expression)